src/Opdex.Platform.WebApi/Controllers/AuthController.cs

```diff
@@ -96,37 +94,4 @@ public class AuthController : ControllerBase
 
         return NoContent();
     }
-
-    /// <summary>Authorize</summary>
-    /// <remarks>Authorizes access to a specific market</remarks>
-    /// <param name="wallet">The wallet public key of the user</param>
-    /// <returns>An access token</returns>
-    [HttpPost("authorize")]
-    public IActionResult Authorize([FromQuery] Address wallet)
```

Comment:
When developing locally, have you come up with an easy way to get the auth token? All I can think of with this now removed is to debug and set a breakpoint when the bearer token is built, then auth with the SASS endpoint

Reply:
Good point, for devnet and up you can rip the bearer token from any api request in the network tab of dev tools. For local, that is not really possible without also running a local client. 

Maybe I put a separate devnet only endpoint back for this purpose only, local/devnet token retrieval

Reply:
https://github.com/Opdex/opdex-v1-api/pull/191